flake.nix

```diff
@@ -120,4 +120,12 @@
   in
     flake-utils.lib.eachDefaultSystem (system: flakeForSystem nixpkgs system);
 }
-# nix-direnv cache busting line: sha256-b/iffKOn7nMiWvM0AIGGzZaJ15NTaBlJff+aja3NQio=
+<<<<<<< HEAD
+# nix-direnv cache busting line: sha256-3NzfNgqSjDILsm2kzP9S0lc+Xmej0gAdDqGJOsWrKdk=
+=======
+<<<<<<< HEAD
+# nix-direnv cache busting line: sha256-3NzfNgqSjDILsm2kzP9S0lc+Xmej0gAdDqGJOsWrKdk=
+=======
+# nix-direnv cache busting line: sha256-3NzfNgqSjDILsm2kzP9S0lc+Xmej0gAdDqGJOsWrKdk=
+>>>>>>> 3bc261cc (tailfs: initial implementation)
+>>>>>>> db634a9b (tailfs: initial implementation)
```

Comment:
these lines seem unintentional, like there was a conflict that needs to be cleaned up.

Reply:
Thanks, fixed!

Reply:
You may be seeing conflict markets in a specific commit, but they're cleaned up in the latest commit.